<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatSimple</class>
 <widget class="QMainWindow" name="chatSimple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chatSimple</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="login_ptn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_ptn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flush_ptn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_ptn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>400</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>651</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>聊天框</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>651</width>
       <height>341</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>241</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>系统消息</string>
    </property>
    <widget class="QTextEdit" name="textEdit_sys">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>241</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(237, 212, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_ptn">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>0</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
